### ISSUE TITLE
unfinished model building?

### ISSUE THREAD

**Issue by helizabeth1103** [google/deepvariant#966]
Hello, 

I've been running model training, and the training is generating model checkpoints, but I am not sure if the format of this output has changed since a previous iteration of deepvariant or if the model did not finish generating custom model files to test. If the model did finish, how do I call the specific checkpoint to test it? If it did not finish, do you have any advice on parameters to adjust to develop a model faster? I allowed it the maximum amount of time my HPC will allow and it timed out. 

Here was the training script I submitted: 

`module load apptainer
apptainer exec --nv /90daydata/pbarc/haley.arnold/AI_Model_Training/Samples/deepvariant_1.6.0.sif /opt/deepvariant/bin/train \
--config=/90daydata/pbarc/haley.arnold/AI_Model_Training/Samples/dv_config.py:base \
--config.train_dataset_pbtxt="shuffledPO1.with_label_channelsize.shuffled.pbtxt" \
--config.tune_dataset_pbtxt="shuffledPO2.with_label_channelsize.shuffled.pbtxt" \
--config.init_checkpoint=gs://deepvariant/models/DeepVariant/1.6.1/checkpoints/wgs/deepvariant.wgs.ckpt \
--config.num_epochs=10 \
--config.learning_rate=0.0002 \
--config.num_validation_examples=0 \
--experiment_dir="model3out/" \
--strategy=mirrored \
--config.batch_size=24`

and here is the contents of the output folder: 

```
checkpoints  
events.out.tfevents.1745308006.atlas-0240.78374.0.v2  
example_info.json
```

here is what is in the checkpoints folder above: 

```
checkpoint  
ckpt-100000.data-00000-of-00001  
ckpt-100000.index
```

and this is what the above 'checkpoint' file says: 

```
model_checkpoint_path: "ckpt-100000"
all_model_checkpoint_paths: "ckpt-100000"
all_model_checkpoint_timestamps: 1746074098.1465378
last_preserved_timestamp: 1745308005.307949
```

thank you for your help!

**Comment by akolesnikov** [comment]
It looks like you had one checkpoint generated - `ckpt-100000`.  You would need to run the training longer for it to converge. Did you run it on a machine with GPU? If so, did you check that GPU was utilized fully? Depending on the amount of data the GPU training can run for up to a week or more, so make sure the timeout is set accordingly. Another option could be to utilize a powerful GPU machine on the cloud.\
You can test the checkpoint by running DeepVariant with `--customized_model` flag pointing to your checkpoint 
```
--customized_model model3out/checkpoints/ckpt-100000
```

More details can be found [here](https://github.com/google/deepvariant/blob/r1.8/docs/deepvariant-training-case-study.md)
